README.md

```diff
@@ -30,4 +30,4 @@ Security issues in the Charmed MongoDB Operator can be reported through [LaunchP
 
 ## Contributing
 
-Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines on enhancements to this charm following best practice guidelines, and [CONTRIBUTING.md](https://github.com/canonical/mongodb-operator/blob/main/CONTRIBUTING.md) for developer guidance.
```

Comment:
I couldn't find the diff here. Maybe is some mistake from GIT itself.

Reply:
yeah I have _no idea_ why it did this, I think this also happened on one of my previous PRs

Reply:
Ah, I saw that as well on the previous PR. I thought I was going crazy :D